net/yt-dlp-ejs/Portfile

```
@@ -0,0 +1,71 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                yt-dlp-ejs
+homepage            https://github.com/yt-dlp/ejs
+version             0.3.1
+
+master_sites        https://files.pythonhosted.org/packages/e7/fd/34fbdaf0d53386c47e219c532a479766cd9336fde34c00834c8e0123df7a/
+distname            yt_dlp_ejs-${version}-py3-none-any
+extract.suffix      .whl
+extract.only
+
+revision            0
+categories          net
+license             public-domain
```

Since you use the wheel here, it also contains code under the ISC and MIT licenses, see the following notice in the README.md of yt-dlp-ejs:

> An exception to this are the prebuilt wheels, which contain both [meriyah](https://github.com/meriyah/meriyah) and [astring](https://github.com/davidbonnet/astring), licensed under [ISC](https://github.com/meriyah/meriyah?tab=ISC-1-ov-file) and [MIT](https://github.com/davidbonnet/astring?tab=MIT-1-ov-file), respectively.

As a consequence, this should be

```suggestion
license             public-domain ISC MIT
```

---

fixed, thanks